scriptmodules/supplementary/bluetooth.sh

```diff
@@ -300,7 +314,7 @@ function boot_bluetooth() {
             while read mac_address; read device_name; do
                 macs+=($mac_address)
             done < <(list_registered_bluetooth)
-            local script="while true; do for mac in ${macs[@]}; do hcitool con | grep -q \"\$mac\" || echo \"connect \$mac\" | bluetoothctl >/dev/null 2>&1; sleep 10; done; done"
+            local script="while true; do for mac in ${macs[@]}; do hcitool con | grep -q \"\$mac\" || { echo \"connect \$mac\nquit\"; sleep 1; } | bluetoothctl >/dev/null 2>&1; sleep 10; done; done"
```

Comment:
Just to follow up, here's the behaviour with BlueZ 5.48:
```
pi@retropie:~/RetroPie-Setup $ echo version | bluetoothctl
[NEW] Controller B8:27:EB:1B:E6:3A retropie [default]
Agent registered
Agent unregistered
[DEL] Controller B8:27:EB:1B:E6:3A retropie [default]
vpi@retropie:~/RetroPie-Setup $
```
Notice that the version string begins to be printed before cutting off. Before I added the "sleep 1" to the end of ```bluez_cmd_bluetooth```, the response challenge was not completing correctly.

Compare to:
```
pi@retropie:~/RetroPie-Setup $ { echo version; sleep 1; } | bluetoothctl
[NEW] Controller B8:27:EB:1B:E6:3A retropie [default]
[NEW] Device C0:33:5E:05:DE:D2 Microsoft Sculpt Comfort Mouse
Agent registered
[Microsoft Sculpt Comfort Mouse]# version
Version 5.48
Agent unregistered
[DEL] Controller B8:27:EB:1B:E6:3A retropie [default]
pi@retropie:~/RetroPie-Setup $
```

Adding the sleep both here and in the (new) dedicated function should increase compatibility with newer BlueZ, without breaking legacy versions.